hbase-server/src/test/java/org/apache/hadoop/hbase/master/assignment/TestAMAssignWithRandExec.java

```diff
@@ -44,10 +46,15 @@ public void testAssignWithRandExec() throws Exception {
 
     rsDispatcher.setMockRsExecutor(new RandRsExecutor());
     // Loop a bunch of times so we hit various combos of exceptions.
```

Comment:
This means of testing annoys me. It's better to explicitly test each response rather than over-test with slop...